opencensus/exporters/trace/ocagent/internal/ocagent_exporter.cc

```diff
@@ -310,7 +310,11 @@ void Handler::InitNode() {
   library_info->set_core_library_version(OPENCENSUS_VERSION);
 
   auto service_info = nodeInfo_.mutable_service_info();
-  service_info->set_name(::opencensus::common::Hostname());
+  if (!opts_.service_name.empty()) {
+    service_info->set_name(opts_.service_name);
+  } else {
+    service_info->set_name(::opencensus::common::Hostname());
```

Comment:
Should we just not set this if not configured? Or this is default for other languages?

Reply:
My reading of the golang exporter is it's set to whatever the user specifies or else empty string. [link](https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/search?q=serviceName&unscoped_q=serviceName)

Ditto [Java](https://github.com/census-instrumentation/opencensus-java/blob/dab6d6f61d4246610a3b10c9c37dbf48bbb96139/exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/OcAgentTraceExporterHandler.java#L59)

I'll remove the Hostname bit.